sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -35,14 +37,9 @@ SELECT
   disqualification_count,
   exposure_count
 FROM
-  `moz-fx-data-shared-prod.telemetry.experiment_enrollment_aggregates_hourly`
+  `moz-fx-data-shared-prod.org_mozilla_fenix_derived.experiment_events_live_v1`
 WHERE
-  window_start > (
-    SELECT
-      MAX(window_end)
-    FROM
-      `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_v1`
-  )
```

Comment:
Keeping this query will cause the materialized view to re-run the underlying query, which is expensive. Using `CURRENT_DATE` prevents it from doing that.